ibm/resource_ibm_private_dns_resource_record.go

```diff
@@ -71,15 +71,13 @@ func resourceIBMPrivateDNSResourceRecord() *schema.Resource {
 			pdnsRecordName: {
 				Type:             schema.TypeString,
 				Required:         true,
-				ForceNew:         true,
-				DiffSuppressFunc: caseDiffSuppress,
+				DiffSuppressFunc: suppressPDNSRecordNameDiff,
 				Description:      "DNS record name",
 			},
 
 			pdnsRecordType: {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
```

Comment:
Name and Record type doesn't seem to have update feature..
Why is force new removed from both attributes?

Reply:
added ForceNew back